### PR TITLE
Handle primitive values in EnumValidationError

### PR DESCRIPTION
**Pull request by P0lip**
I feel like I could organize these test fixtures slightly better, so I'm all ears if you have any idea on how to do it any better.

Moreover, I suspect we could add an extra check in `findBestMatch` method, but I assumed Ajv will never report an error with dataPath for non-object input.

### CODE REVIEW

**Comment by torifat** [comment]
Hey, thanks for your PR and these additional test cases :)

Currently, I'm converting the whole project to TypeScript. And, it's almost done -> https://github.com/torifat/better-ajv-errors/tree/typescript

I will probably focus on organising tests after that. Feel free to share your ideas.

Personally, I don't like snapshot testing for everything but I initially did it to save time. So, my plan is to move some to them to more approachable unit tests and only use snapshot for like like CLI output or big chunk of data validation.

**Comment by torifat** [comment]
Can you please run `yarn changeset` (https://asciinema.org/a/270002) and then commit those files too. It will automatically release a new package in `npm` in that case.

**Comment by P0lip** [comment]
@torifat 
Done!

> Currently, I'm converting the whole project to TypeScript. And, it's almost done -> https://github.com/torifat/better-ajv-errors/tree/typescript

That's great news.

> Personally, I don't like snapshot testing for everything but I initially did it to save time. So, my plan is to move some to them to more approachable unit tests and only use snapshot for like like CLI output or big chunk of data validation.

I don't like snapshots either, I tend not to use them at all.
They are still great to prevent regressions, though, so it's certainly better to have snapshots than nothing.

File: .changeset/tiny-starfishes-scream/changes.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "releases": [{ "name": "better-ajv-errors", "type": "minor" }],
```
Comment (torifat) [inline .changeset/tiny-starfishes-scream/changes.json]: Sorry, can you please change it to `patch`. Since we are still on `0.x.x`.

Comment (P0lip) [reply .changeset/tiny-starfishes-scream/changes.json]: Ah, sorry, I meant to release patch obviously. Changing it

**Pull request merged**